src/compiler/scala/tools/nsc/Properties.scala

```diff
@@ -28,9 +28,6 @@ object Properties extends scala.util.PropertiesTrait {
   // a newline so as not to break the user's terminal)
   def shellInterruptedString = scalaPropOrElse("shell.interrupted", f":quit$lineSeparator")
 
-  // derived values
-  def isEmacsShell         = propOrEmpty("env.emacs") != ""
```

Comment:
This may be used elsewhere. Juuuuust to be remain on the safe side, let's deprecate instead of removing.